docs/pages/router/reference/api-routes.mdx

```diff
@@ -282,3 +282,85 @@ Start the server with `node` command:
 <Terminal cmd={['$ node server.js']} />
 
 </Step>
+
+### Netlify
+
+> This is experimental and subject to breaking changes. We have no continuous tests against this configuration.
+
+<Step label="1">
+
+Create a server entry file, all requests will be delegated through this middleware. The exact file location is important.
+
+```js netlify/functions/server.js
+const { createRequestHandler } = require('@expo/server/adapter/netlify');
+
+const handler = createRequestHandler({
+  /* @info Points to the root `dist/` (output) folder */
+  build: require('path').join(__dirname, '../../dist'),
+  /* @end */
+  mode: process.env.NODE_ENV,
+});
+
+module.exports = { handler };
+```
+
+</Step>
+
+<Step label="2">
+
+Create a Netlify configuration file to redirect all requests to the server function.
+
+```yaml netlify.toml
+[build]
+  command = "expo export -p web"
+  functions = "netlify/functions"
+  publish = "dist"
+
+[[redirects]]
+  from = "/*"
+  to = "/.netlify/functions/server"
+  status = 404
+
+[functions]
+  # Include everything to ensure dynamic routes can be used.
+  included_files = ["dist/**/*"]
+
+[[headers]]
+  for = "/dist/_expo/functions/*"
+  [headers.values]
+    # Set to 60 seconds as an example.
+    "Cache-Control" = "public, max-age=60, s-maxage=60"
+```
+
+</Step>
+
+<Step label="3">
+
+After you have created the configuration files, you can build the website and functions with Expo CLI:
+
+<Terminal cmd={['$ npx expo export -p web']} />
+
+</Step>
+
+<Step label="4">
```

Comment:
In step 4, can we redirect developers to follow the steps here: https://docs.expo.dev/distribution/publishing-websites/#netlify. Or the steps defined in the link are not relevant here?

Reply:
Those steps will need to be updated in the future, they should indicate that you use `--dir` with server mode.